lib/History/History.js

```diff
@@ -158,6 +159,17 @@ var History = module.exports = kind.singleton({
 	},
 
 	/**
+	* @private
+	*/
+	enableBackHistoryAPIChanged: function () {
+		if (this.enableBackHistoryAPI) {
+			window.onpopstate = this.bindSafely(function() {this.popStateHandler();});
```

Comment:
I don't think we want to assign the event handler like this. `onpopstate` is already included in the enyo dispatcher so we could use Signals here. We could also use `dispatcher.listen()` to add the listener. Either would be okay, imo.